src/core/matchers/toHaveClasses.js

```diff
@@ -0,0 +1,36 @@
+getJasmineRequireObj().toHaveClasses = function(j$) {
+  /**
+   * {@link expect} the actual value to be a DOM element that has the expected classes
+   * @function
+   * @name matchers#toHaveClasses
+   * @since 5.6.0
+   * @param {Object} expected - The class names to test for
+   * @example
+   * const el = document.createElement('div');
+   * el.className = 'foo bar baz';
+   * expect(el).toHaveClasses(['bar', 'baz']);
+   */
+  function toHaveClasses(matchersUtil) {
+    return {
+      compare: function(actual, expected) {
+        if (!isElement(actual)) {
+          throw new Error(matchersUtil.pp(actual) + ' is not a DOM element');
+        }
+
+        return {
+          pass: expected.every(e =>
+            actual.classList.value.split(' ').some(c => c === e)
```

Comment:
Looks good for the most part but couldn't this line just be `actual.classList.contains(e)`?

Reply:
Oh yeah! Must've gotten too deep into the weeds and completely missed the obvious.